src/pages/user/Settings.js

```diff
@@ -106,6 +117,21 @@ export default class UserSettings extends Component {
               </InputGroup>
             </Col>
           </Row>
+          <Row className="text-center">
+            <Col md="12" className="align-self-center">
+              Toggle Dark Mode
+              <ToggleButton
+                value={this.props.theme === 'default' ? false : true}
+                onClick={() => this.props.actions.toggleDarkMode()}
```

Comment:
Would you mind destructuring `this.props.actions` at the top of the render?

Reply:
Sure.